projects/plugins/crm/includes/class-oauth-handler.php

```diff
@@ -3,7 +3,7 @@
  * Jetpack CRM
  * https://jetpackcrm.com
  *
- * OAuth connection handler. Requires PHP 7.3+
```

Comment:
Good change since CRM has long since been 7.4-minimum, but not related to the dropping of PHP 7.2.

Reply:
Updated PR title.